test/features/sdk_flow.feature

```diff
@@ -0,0 +1,203 @@
+    Given I navigate to the SDK
+    When I click on verify_identity (SDK)
+    And I click on passport ()
+    Then page_title () should contain "Upload the front of your document"
+    When I upload llama <type> on file_upload ()
+    Then I should see uploaded_<type>image ()
+    And confirmation_text () should contain "Please confirm that you are happy with this photo."
+    When I click on confirm ()
+    Then error_message () should contain "No document detected"
+    And error_instruction () should contain "Make sure all the document is in picture"
+
+    Examples:
+      | type |
+      |      |
+      | pdf  |
+
+  Scenario: I should not be able to upload a document over 10MB.
+    Given I navigate to the SDK
+    When I click on verify_identity (SDK)
+    And I click on passport ()
+    Then page_title () should contain "Upload the front of your document"
+    When I upload over_10mb_face on file_upload ()
+    Then upload_error_message () should contain "File size too large. Size needs to be smaller than 10MB."
+
+  Scenario: I should not be able to upload an image of a face over 10MB.
+    Given I navigate to the SDK
+    When I click on verify_identity (SDK)
+    And I click on passport ()
+    Then page_title () should contain "Upload the front of your document"
+    When I upload passport on file_upload ()
+    Then I should see uploaded_image ()
+    And confirmation_text () should contain "Please confirm that you are happy with this photo."
+    When I click on confirm ()
+    Then page_title () should contain "Upload a picture of your face"
+    When I upload over_10mb_face on file_upload ()
+    Then upload_error_message () should contain "File size too large. Size needs to be smaller than 10MB."
+
+  Scenario: I should not be able to upload an unsupported type image of a face
+    Given I navigate to the SDK
+    When I click on verify_identity (SDK)
+    And I click on passport ()
+    Then page_title () should contain "Upload the front of your document"
+    When I upload passport on file_upload ()
+    Then I should see uploaded_image ()
+    And confirmation_text () should contain "Please confirm that you are happy with this photo."
+    When I click on confirm ()
+    Then page_title () should contain "Upload a picture of your face"
+    When I upload one_face pdf on file_upload ()
+    Then I should see uploaded_pdfimage ()
+    When I click on confirm ()
+    Then error_message () should contain "Unsupported file type"
+    And error_instruction () should contain "Try using a .jpg or .png file"
+
+  Scenario: I should not be able to upload an image containing multiple faces
+    Given I navigate to the SDK
+    When I click on verify_identity (SDK)
+    And I click on passport ()
+    Then page_title () should contain "Upload the front of your document"
+    When I upload passport on file_upload ()
+    Then I should see uploaded_image ()
```

Comment:
Is it possible to group steps to avoid repetitions? For example, could you group the steps from
`When I click on verify_identity (SDK)` to `Then I should see uploaded_image ()` in one step called "When I successfully upload a passport" ?

Reply:
Absolutely! I was thinking of that too as there are many repetitive pieces of code. Though I would rather do it as part of other PR